docs/collect-offline-maps.rst

```diff
@@ -1,67 +1,122 @@
+
+If your Collect users are remote and don't have access to file sharing services, you can send them a direct link to a file or the file itself via WhatsApp, email, etc. They will then save the file to their Downloads and import it from Collect.
+
+If you are setting up multiple devices from a computer, you may want to :ref:`manually place layer files in the correct directory <transferring-offline-tiles-manually>`.
+
+.. _adding-offline-layers:
+
+Adding offline layers to Collect
+---------------------------------
+
+.. note::
+  
+  You can also :ref:`manually place layer files in the correct directory <transferring-offline-tiles-manually>`.
+
+You can add offline layers to Collect from the :ref:`mapping-settings` or by tapping on the layers button (:fa:`layer-group`) directly from a map. Both options give you access to the same menu which lets you configure the offline layer for all maps in the current Collect project.
+
+.. image:: /img/collect-offline-maps/select-offline-layer.png
+  :class: device-screen-vertical
+
+When you tap the :guilabel:`Add layers` button, you will see a file picker. To import a single file, tap on it and you will go back to Collect to finish the import process. If you need to import multiple offline layers, you can select multiple files. This process will vary a bit by Android version and device type but it is usually done by tapping on their file type icon. After selecting multiple files, a select button will appear at the top or bottom right. Tap that button to confirm your selection and return to Collect.
+
+Once you have selected the layer(s) that you want to import, you will see a confirmation screen showing the layer names and options for sharing the layer(s) between projects or not. This setting is only relevant if you use multiple Collect projects. If you do use multiple projects and they use different offline layers, it can be helpful to import a different list in each individual project. Additionally, when layers are imported at the project level, they will be deleted when the project is deleted.
+
+.. image:: /img/collect-offline-maps/layer-access.png
+  :class: device-screen-vertical
+
+Once you confirm that you want to add the layer(s), you will see the newly added layer(s) in the layer selection list. The list shows the ``name`` specified by each MBTiles file. You can also see an offline layer's filename by tapping on its name to expand. This can be helpful if the filename is more descriptive than the embedded name.
+
+When Collect imports MBTiles files from device storage, the files are copied, not moved. After they have been imported, it is safe to delete the source files from device storage, usually the Downloads folder, to free up space. Alternately, you can keep source files and delete offline layers from Collect when they are no longer needed. To delete offline layer files, tap on the layer name to see an expanded view that includes a delete button.
```

Comment:
s/Alternately/Alternatively/ Do this globally.

Reply:
Yikes!